addons/medical_treatment/functions/fnc_canTreat.sqf

```diff
@@ -27,6 +27,14 @@ isClass _config
 && {_medic != _patient || {GET_NUMBER_ENTRY(_config >> "allowSelfTreatment") == 1}}
 && {[_medic, GET_NUMBER_ENTRY(_config >> "medicRequired")] call FUNC(isMedic)}
 && {
+    // Handle holster settings
+    GVAR(holsterRequired) == 0
+    || !(vehicle _medic isEqualTo _medic) // medic is in a vehicle, so weapon is considered holstered
+    || !(vehicle _patient isEqualTo _patient) // patient is in a vehicle, ^
```

Comment:
A simple != might be faster than a ! and isEqualTo.

Reply:
Ah ok, from what I'd read on the wiki it sounded like isEqualTo might've been faster. Will correct now.

Reply:
isEqualTo is a TINY bit faster than ==.
But ! (x isEqualTo y) are more script commands and is slower.